### ISSUE TITLE
SIREN not converge

### ISSUE THREAD

**Issue by zhangmozhe** [vsitzmann/siren#19]
Hi, thanks for this great work. I recently adopted SIREN in my problem, but during training the loss becomes divergent. I've set the omega to 30 by default. Could you please give suggestion about this. Thanks!

![image](https://user-images.githubusercontent.com/10203551/94515028-3cc5be00-0255-11eb-9f89-22eb99f2ec23.png)

**Comment by zhangmozhe** [comment]
I have solved the issue by normalizing the input to (-1,1) and reduce the learning rate to 1e-5. It seems that normalizing to (-1,1) is very important when using sine activation.
